process/track/interface.go

```diff
@@ -0,0 +1,33 @@
+package track
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data"
+)
+
+type blockNotarizerHandler interface {
```

Comment:
Suggestion to keep the interface names unexported but the functions exported in order to be able to stub these interfaces in a separate package (mock). Can be left of the next PR when unit tests will be written.

Reply:
ok